gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/sampling/RandomVertexNeighborhoodSamplingTest.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.operators.sampling;
+
+import com.google.common.collect.Lists;
+import org.apache.flink.api.java.io.LocalCollectionOutputFormat;
+import org.gradoop.common.model.impl.id.GradoopId;
+import org.gradoop.common.model.impl.pojo.Edge;
+import org.gradoop.common.model.impl.pojo.Vertex;
+import org.gradoop.flink.model.GradoopFlinkTestBase;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.model.impl.operators.sampling.functions.Neighborhood;
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class RandomVertexNeighborhoodSamplingTest extends GradoopFlinkTestBase {
```

Comment:
Since you always validate your result with the same method you could try to implement a parameterized test. See  [BaseClass of Parameterized Test](https://github.com/dbs-leipzig/gradoop/blob/master/gradoop-flink/src/test/java/org/gradoop/flink/algorithms/fsm/transactional/basic/BasicPatternsTransactionalFSMTestBase.java) and [Call Class](https://github.com/dbs-leipzig/gradoop/blob/master/gradoop-flink/src/test/java/org/gradoop/flink/algorithms/fsm/transactional/basic/BasicPatternsDIMSpanTest.java)

Your Sampling methods would be called with @Parameterized.Parameters and tested against a validateGraph test method.